File: Iniciante/1037 - Intervalo.py
"""
Autor: Pierre Vieira
Data da submissão: 22/02/2019 17:47:29
"""
numero = float(input())
if 0 <= numero <= 25:
    print("Intervalo [0,25]")
elif 25 < numero <= 50:
    print("Intervalo (25,50]")
elif 50 < numero <= 75:
    print("Intervalo (50,75]")
elif 75 < numero <= 100:
    print("Intervalo (75,100]")
else:
    print("Fora de intervalo")

File: C#/Iniciante/1037 - Intervalo.cs
/*
* Autor: Pierre Vieira
* Data de submissão: 14/05/2020 10:51:31
*/
using System;
using System.Globalization;

namespace URI {
    class URI {
        static void Main(string[] args)
        {
            float numero;
            numero = float.Parse(Console.ReadLine(), CultureInfo.InvariantCulture);
            if (numero >= 0 && numero <= 25)
            {
                Console.WriteLine("Intervalo [0,25]");
            }
            else if (numero > 25 && numero <= 50)
            {
                Console.WriteLine("Intervalo (25,50]");
            }
            else if (numero > 50 && numero <= 75)
            {
                Console.WriteLine("Intervalo (50,75]");
            }
            else if (numero > 75 && numero <= 100)
            {
                Console.WriteLine("Intervalo (75,100]");
            }
            else
            {
                Console.WriteLine("Fora de intervalo");
            }
        }
    }
}
